test/remote/gateways/remote_migs_test.rb

```diff
@@ -104,6 +104,16 @@ def test_refund
     # assert_equal 'Approved', response.message
```

Comment:
The remote test for the `refund` action was previously commented out and labelled as "not working." I uncommented out this test and confirmed that it still isn't working. Should I attempt to resolve this problem or simply forego writing a remote test for `refund` passing `tx_source`?

Reply:
Since the comment says that refunds were not working in the test environment, it sounds like we likely won't be able to do anything to fix it, as we are indeed limited by the environment settings. Foregoing the remote test sounds fine.